index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal-Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+</head>
+<body>
+  <header class="header">
+    <nav class="top-nav">
+      <div class="logo">
+        <p>
+          my logo
+        </p>
+      </div>
+      <nav class="nav-items">
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </nav>
+    <section class="text">
+      <header class="text-section">
+        <h1 class="head-section">
+          I&#8217;m Adeka Glad to see you!
```

Comment:
- [ ] nice job. please kindly add a background image not only to make it look like the figma version, but also to make it look beautiful and presentable. nice work

yours
![Screenshot (741)](https://user-images.githubusercontent.com/65298048/212117214-f1d854b5-5d3f-47e3-87f6-8db7bf7da1a8.png)
figma version
![Screenshot (742)](https://user-images.githubusercontent.com/65298048/212117464-ae4aea5f-aeca-44a0-bd35-6f2150ed327b.png)